TwitchDownloaderCore/VideoPlatforms/Twitch/ChatRoot.cs

```diff
@@ -273,5 +274,6 @@ public class ChatRoot
         public Video video { get; set; }
         public List<Comment> comments { get; set; }
         public EmbeddedData embeddedData { get; set; }
+        public VideoPlatform videoPlatform { get; set; } = VideoPlatform.Twitch;
```

Comment:
Using an enum results in an integer rather than a string in the JSON file. If it's alright, I would like to change that to a string via a custom [JsonConverter](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/converters-how-to?pivots=dotnet-7-0), but it would break any existing Kick chats.

Reply:
Go for it, I don't care about breaking kick chats until this thing is actually released